packages/iov-bns/src/bnsconnection.spec.ts

```diff
@@ -1104,6 +1111,63 @@ describe("BnsConnection", () => {
       connection.disconnect();
     });
 
+    it("any account can return an escrow", async () => {
```

Comment:
This description makes it sound like anyone can cancel anyone else's escrow.

Reply:
This is exactly what happens, but only after timeout. Any idea for better wording? I wanted to pick up the verb from the transaction name

Reply:
Ah yeah, I remember now. Perhaps just "any account can return an escrow (after the timeout)".

Reply:
This behaviour is of critical importance when we want to use escrows for vesting, which after the lockup period send to a Ledger address. This Ledger address can only sign send transactions, so "a friend" is needed to help out triggering the returns.